hdate/zmanim.py

```diff
@@ -59,9 +59,9 @@ def __init__(
         calculations.
         """
         self.location = location
-        self.hebrew = hebrew
-        self.candle_lighting_offset = candle_lighting_offset
-        self.havdalah_offset = havdalah_offset
+        self.language: str = language
+        self.candle_lighting_offset: int = candle_lighting_offset
+        self.havdalah_offset: int = havdalah_offset
```

Comment:
Not needed

Reply:
When removing : int I have many error "no-untyped-def"